### PR TITLE
<config/docker>: use cloud sql in place of container-based postgres

### PR DESCRIPTION
**Pull request by david30907d**
check the detail of our cloud sql instance here: https://console.cloud.google.com/sql/instances/dev-team-db-for-web-data-infra/overview?authuser=0&cloudshell=true&hl=zh-tw&project=pycontw-225217&supportedpurview=project

### CODE REVIEW

File: data_team/metabase_server/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [Metabase](https://www.metabase.com/) is an open-source business intelligence tool adopted by PyCon TW Data Team to track cross-year conference data. The docker-compose script handles four services:
 
 - `metabase`: The Metabase server.
-- `postgres`: The database storing the data used by the Metabase server.
+    - `postgres`: The database storing the data used by the Metabase server. Please get the public IP of our GCP cloud SQL and place it into [docker-compose.yml](./docker-compose.yml) 's `MB_DB_HOST`
```
Comment (david30907d) [inline data_team/metabase_server/README.md:6]: updated instruction here

**Comment by david30907d** [comment]
@mattwang44

File: data_team/metabase_server/docker-compose.yml [hunk 2]
```diff
@@ -48,8 +37,8 @@ services:
       - MB_REDIRECT_ALL_REQUESTS_TO_HTTPS=false
       - MB_SITE_URL=null
       - MB_DB_DBNAME=metabase
-      - MB_DB_HOST=postgres
+      - MB_DB_HOST=<get the public IP from GCP Cloud SQL's db instance>
```
Comment (mattwang44) [inline data_team/metabase_server/docker-compose.yml]: How about taking env vars from `.env`? May need to modify the instruction in the readme and add a new key in `.env.example` (and delete the old one) as well.
```suggestion
      - MB_DB_HOST=${CLOUD_SQL_URI}  # rename it
```

File: data_team/metabase_server/README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 [Metabase](https://www.metabase.com/) is an open-source business intelligence tool adopted by PyCon TW Data Team to track cross-year conference data. The docker-compose script handles four services:
```
Comment (mattwang44) [inline data_team/metabase_server/README.md]: nit
```suggestion
[Metabase](https://www.metabase.com/) is an open-source business intelligence tool adopted by PyCon TW Data Team to track cross-year conference data. The docker-compose script handles several services, as listed below:
```

File: data_team/metabase_server/docker-compose.yml (hunk 2 above)
Comment (mattwang44) [reply data_team/metabase_server/docker-compose.yml]: @david30907d would you take care of this one? or you'd like to remove the `.env`?

Comment (david30907d) [reply data_team/metabase_server/docker-compose.yml]: oh sorry for the late reply, will do it tmrw~

Comment (david30907d) [reply data_team/metabase_server/docker-compose.yml]: done~

File: data_team/metabase_server/docker-compose.yml [hunk 4]
```diff
@@ -48,7 +37,7 @@ services:
       - MB_REDIRECT_ALL_REQUESTS_TO_HTTPS=false
       - MB_SITE_URL=null
       - MB_DB_DBNAME=metabase
-      - MB_DB_HOST=postgres
+      - MB_DB_HOST=${CLOUD_SQL_IP}
```
Comment (david30907d) [inline data_team/metabase_server/docker-compose.yml:40]: rename it into `CLOUD_SQL_IP`

File: data_team/metabase_server/.env.example [hunk 5]
```diff
@@ -1,2 +1,3 @@
 POSTGRES_USER=
-POSTGRES_PASSWORD=
\ No newline at end of file
+POSTGRES_PASSWORD=
+CLOUD_SQL_IP=<get the public IP from GCP Cloud SQL's db instance>
```
Comment (david30907d) [inline data_team/metabase_server/.env.example:3]: and put my comment here 😄

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**